easybuild/easyconfigs/p/PGPLOT/PGPLOT-5.2.2-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'CmdCp'
+
+name = 'PGPLOT'
+version = '5.2.2'
+
+homepage = 'https://sites.astro.caltech.edu/~tjp/pgplot/'
+description = """The PGPLOT Graphics Subroutine Library is a Fortran- or C-callable,
+device-independent graphics package for making simple scientific graphs. It is intended
+for making graphical images of publication quality with minimum effort on the part of
+the user. For most applications, the program can be device-independent, and the output
+can be directed to the appropriate device at run time."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['ftp://ftp.astro.caltech.edu/pub/%(namelower)s/']
+sources = ['%(namelower)s522.tar.gz']
```

Comment:
Previous EasyConfigs have `sources = ['%%(namelower)s%s.tar.gz' % version.replace('.', '')] `, could you replace with that? it is slightly more generic, and will work with EasyBuilds `--try-software-version` option, which is a nice-to-have.